--- libkml-0.6.1.org/configure.ac	2009-04-13 09:24:15.041537747 +0530
+++ libkml-0.6.1/configure.ac	2009-04-13 09:25:24.670599502 +0530
@@ -72,8 +72,8 @@
 	AC_CHECK_PROG(SWIG, swig, swig)
 	if test "x$SWIG" != x; then
 		SWIG_VERSION=`swig -version 2>&1|grep Version|cut -d" " -f3`
-		if test $SWIG_VERSION != 1.3.35; then
-			AC_MSG_WARN([You have SWIG $SWIG_VERSION installed, but libkml requires SWIG 1.3.35. The bindings will not be built.])
+		if test $SWIG_VERSION >= 1.3.35; then
+			AC_MSG_WARN([You have SWIG have $SWIG_VERSION installed, but libkml requires greater then SWIG 1.3.35. The bindings will not be built.])
 			SWIG=
 		fi
 	fi
